pyquil/quil.py

```diff
@@ -35,6 +33,9 @@
                              Jump, Label, JumpConditional, JumpTarget, JumpUnless, JumpWhen,
                              Declare, Halt, Reset, ResetQubit, DefPermutationGate)
 
+# for typing purpose
```

Comment:
In other files, the typing imports are not grouped together that it may cause confusion.

Reply:
I can see the value in grouping typing-related imports, but on the other hand this breaks [the PEP8 convention](https://www.python.org/dev/peps/pep-0008/#imports) of ordering imports as 1) standard lib imports, 2) thrid-party imports, 3) application-specific imports.

Reply:
The order that is used in Zulip project is 1. typing imports, 2., 3., 4., ... PEP8 convention imports (e.g. https://github.com/zulip/zulip/blob/master/zerver/models.py).

Reply:
PEP8 doesn't specify the order of typing imports unfortunately.

Reply:
lets just do
> 1) standard lib imports, 2) third-party imports, 3) application-specific imports

Reply:
OK, so you meant, no need to make separate section that is specific to typing imports.

Reply:
Fixed in 6th commit.